### DISCUSSION TITLE
Build fail :spi:common:core-spi:test FAILED

### DISCUSSION THREAD

**Discussion by tnolden** [eclipse-edc/Connector#2443] [Q&A]
Hey,
I am trying to set up the current version. However, I get an error during the tests:

![image](https://user-images.githubusercontent.com/119170387/215543906-f6e9f60d-b49e-4ad9-a113-38a2976a6c9d.png)

1 test failed: VaultCertificateResolverTest.resolveCertificate

Error: Found certificate with id [key], but failed to convert it

I hope you can help me.

**Comment by tnolden** [comment]
I stil get the error, when i'm trying to call the main build file of the connector:

![error](https://github.com/eclipse-edc/Connector/assets/119170387/29b61af8-5edd-4058-b776-06e03e935c31)

Any suggestions why the test fails?

**Reply by ndr-brt** [reply]
there's some discussion here, OS issue: https://github.com/eclipse-edc/Connector/issues/2596
